jupyterhub/playbook.yml

```diff
@@ -206,6 +206,14 @@
       read_timeout: 86400
       host_header: "$host"
 
+  handlers:
```

Comment:
This server is currently using certbot to automatically renew lets-encrypt certificates. The `ssl-certificate` role is only used to generate self-signed certificates for testing. There's no harm having this here, but in practice it'll never be used unless we get rid of lets-encrypt on the training/outreach servers.